src/towncrier/create.py

```diff
@@ -65,21 +76,44 @@ def _main(
     * .removal - a deprecation or removal of public API,
     * .misc - a ticket has been closed, but it is not of interest to users.
     """
-    __main(ctx, directory, config, filename, edit, content)
+    __main(ctx, directory, config, filename, edit, content, eof_newline)
 
 
 def __main(
     ctx: click.Context,
     directory: str | None,
     config_path: str | None,
     filename: str,
-    edit: bool,
+    edit: bool | None,
     content: str,
+    eof_newline: bool | None,
 ) -> None:
     """
     The main entry point.
     """
     base_directory, config = load_config_from_options(directory, config_path)
+    if eof_newline is None:
+        eof_newline = config.create_eof_newline
```

Comment:
Just trying to understand what is going on here 

```suggestion
    if eof_newline is None:
        # If no explicit flag was provided via CLI
        # default to the value from the configuration file.
        eof_newline = config.create_eof_newline
```

Reply:
Flag gone now